fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -218,7 +251,14 @@ private LogicalPlan visitCommonSelectQueryClausePlan(
             withProject = withFilter;
         }
 
-        return withProject;
+        LogicalPlan withAgg;
+        if (aggClause != null) {
```

Comment:
we should do with project only when aggClause is null

Reply:
done